packages/governance/README.md

```diff
@@ -0,0 +1,218 @@
+# Governance
+
+This package provides Registrars and VoteCounters to create a general
+framework for governance. It has implementations for particular kinds of
+electorates and different ways of tallying votes.
+
+The registrars and VoteCounters are self-describing and reveal what they are
+connected to so that voters can verify that their votes mean what they say and
+will be tabulated as expected.
+
+Any occasion of governance starts with the creation of a Registrar. Two kinds
```

Comment:
@Chris-Hibbert writes in https://github.com/Agoric/agoric-sdk/issues/3869#issuecomment-930332179
> I wrote a note-to-self on finding something public-facing to rename to be the electorate.

I'm putting it here so that maybe github will help us track it.